src/cmd/builtin/dump.c

```diff
@@ -29,6 +29,24 @@
 
```

Comment:
Commit message:
> Rate cap the lookups at 1000 RPCs.

This isn't really a rate cap - it's a concurrency cap, right?